djredcap/management/subcommands/inspect.py

```diff
@@ -58,14 +58,21 @@ def handle(self, fileName=None, *args, **options):
         fin = open(fileName,'rU')
         dialect = csv.Sniffer().sniff(fin.read(1024))
         fin.seek(0)
-        reader = csv.DictReader(fin, fieldnames=header_keys, dialect=dialect)
+<<<<<<< HEAD
```

Comment:
Looks like you did not clean up the conflict. These breakpoints separate the lines corresponding to the conflicting files. Go through and remove these lines and the code that is incorrect.